tarpc/src/lib.rs

```diff
@@ -255,7 +255,6 @@ use std::{any::Any, error::Error, io, sync::Arc, time::Instant};
 /// A message from a client to a server.
 #[derive(Debug)]
 #[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize))]
-#[non_exhaustive]
```

Comment:
Was this intentional? Making these types exhaustive commits tarpc to never extending the rpc protocol. I don't think we want to make that commitment.

Reply:
Ah I see — yeah for now, can we separate it out to a separate PR? There may be other options we can consider as well, such as a default constructor.